README.md

```diff
@@ -1,30 +1,36 @@
 # `loco-openapi-initializer`
+
 This crate adds OpenAPI support to Loco by using a initializer.
 
 The Loco OpenAPI integration is generated using [`Utoipa`](https://github.com/juhaku/utoipa)
 
 # Installation
 
 ## `Cargo.toml`
+
 Edit your `Cargo.toml` file
 
 Add the `loco-openapi` initializer, with one or multiple of the following features flags:
+
 - `swagger`
 - `redoc`
 - `scalar`
 - `full`
 
 ### Example
+
 ```toml
 # Cargo.toml
 [dependencies]
 loco-openapi = { version = "*", features = [
     "full",
-], git = "https://github.com/loco-rs/loco-openapi-Initializer", branch = "master" }
+], git = "https://github.com/loco-rs/loco-openapi-Initializer", branch = "main" }
```

Comment:
Why did you change it to the main branch? At Loco, we’re using the master branch.